index.bs

```diff
@@ -115,7 +118,16 @@ spec: webidl
       The UA may collect information about a user's intentions in any way its
       authors believe is appropriate. This information can come from explicit
       user action, aggregate behavior of both the relevant user and other users,
-      or other sources this specification hasn't anticipated.
+      or <a>implicit signals</a> this specification hasn't anticipated.
+      <div class="note">
+        The <dfn>implicit signals</dfn> could be, for example, the
+        <a>install</a> status of a web application, frequency and recency of
+        visits. A user that has installed a web application and used it
+        frequently and recently is more likely to have established a trust
+        relationship with the application provider. Implementations are
+        advised to exercise caution when implementing strategies that rely on
```

Comment:
s/are advised to exercise caution/should be cautious/?

Reply:
I want to avoid using RFC 2119 terms in information notes, so I'll will leave this as is.

Reply:
SG.